airbyte-cdk/java/airbyte-cdk/dependencies/src/main/kotlin/io/airbyte/commons/io/IOs.kt

```diff
@@ -13,6 +13,7 @@ import java.util.*
 import org.apache.commons.io.input.ReversedLinesFileReader
 
 object IOs {
```

Comment:
This whole class should be just companion object.

Reply:
this is not a class, this is an object. As such, it's its own companion object. So everything in it can be declared @JvmStatic